test/api/v1/routes/exercises_test.rb

```diff
@@ -63,6 +63,28 @@ def test_restore_exercises_and_solutions
     end
   end
 
+  def test_get_exercises_by_track
+    VCR.use_cassette('exercism_api_error') do
+      get '/v2/exercises/invalid_track', key: 'abc123'
+      options = { format: :json, name: 'get_exercises_invalid_track_error' }
+      Approvals.verify(last_response.body, options)
+    end
+  end
+
+  def test_get_exercises_by_track_slug
+    VCR.use_cassette('exercism_api_error') do
+      get '/v2/exercises/invalid_track/apple', key: 'abc123'
+      options = { format: :json, name: 'get_exercises_invalid_track_error' }
+      Approvals.verify(last_response.body, options)
+    end
+
+    VCR.use_cassette('exercism_api_error') do
```

Comment:
I don't think we need this VCR either since `/v2/exercises/:track/:slug` doesn't seem to be making any external calls. Could you verify this?